lbz/request.py

```diff
@@ -33,21 +33,21 @@ def __init__(
         query_params: dict = None,
         user: User = None,
     ):
-        self.query_params = MultiDict(query_params)
-        self.headers = headers
-        self.uri_params = uri_params
-        self.method = method
-        self.context = context
+        self.query_params: MultiDict = MultiDict(query_params or {})
```

Comment:
You don't have to set type if you are initializing this variable. It is obvious it will be `MultiDict` 😉